e2e-tests/playwright/utils/UIhelper.ts

```diff
@@ -182,6 +212,12 @@ export class UIhelper {
     await expect(link).toBeVisible();
   }
 
+  async textLocator(selector: string, text: string) {
```

Comment:
Let's not take the selector as a parameter. We already have a generic method, `uiHelper.verifyText()`, for text verification. If you want to validate text in an MuiAlert, perhaps you could rename this method to `verifyMuiAlert(text: string)` and remove the selector parameter. For verifications specific to that page, it would be more appropriate to create a method in `support/pages/rbac.ts`.